Formula/patchelf.rb

```diff
@@ -0,0 +1,25 @@
+class Patchelf < Formula
+  desc "Modify dynamic ELF executables"
+  homepage "https://nixos.org/patchelf.html"
+  url "https://nixos.org/releases/patchelf/patchelf-0.9/patchelf-0.9.tar.gz"
```

Comment:
Development is done on GitHub https://github.com/NixOS/patchelf, should we use that as tarball source in order for updates to be detected automatically by livecheck?

Reply:
No we want to use the canonical url. The livecheck can be customized as needed.

Reply:
In which tap does `brew livecheck` live?

Reply:
https://github.com/Homebrew/homebrew-livecheck